circle.yml

```diff
@@ -70,7 +78,12 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-
+      - run:
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
This will run `aws-nuke` every time we do a release. We probably only want to do it nightly, right?

Reply:
Thanks for catching this, left over from a copy paste